biome.jsonc

```
@@ -0,0 +1,12 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.6.4/schema.json",
```

```suggestion
	"$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
```

Refs: https://biomejs.dev/reference/configuration/